<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanas</class>
 <widget class="QMainWindow" name="ventanas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1365</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="Game_window" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>381</width>
      <height>311</height>
     </rect>
    </property>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>271</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="L_nivel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>nivel: 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="L_vidas">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>vidas</string>
     </property>
    </widget>
    <widget class="QLabel" name="L_puntos">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>puntos: 0</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Game_window_2" native="true">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>411</width>
      <height>311</height>
     </rect>
    </property>
    <widget class="QGraphicsView" name="graphicsView_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>271</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;EDAD DE ORO&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="backgroundLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>271</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>jugar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>registrarte</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Game_window_3" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>331</width>
      <height>301</height>
     </rect>
    </property>
    <widget class="QGraphicsView" name="graphicsView_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>271</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>REGISTRO</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>usuario:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>contrase√±a:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>230</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Iniciar Sesion</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Game_window_4" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>381</width>
      <height>311</height>
     </rect>
    </property>
    <widget class="QGraphicsView" name="graphicsView_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>271</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="imageLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>271</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Game_window_5" native="true">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>340</y>
      <width>381</width>
      <height>311</height>
     </rect>
    </property>
    <widget class="QGraphicsView" name="graphicsView_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>271</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Game_window_6" native="true">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>690</y>
      <width>381</width>
      <height>311</height>
     </rect>
    </property>
    <widget class="QGraphicsView" name="graphicsView_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>271</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
